src/develop/pixelpipe_hb.c

```diff
@@ -2710,10 +2710,10 @@ gboolean dt_dev_pixelpipe_process(
                " opencl for this session!\n");
       dt_control_log
         (_("OpenCL errors encountered; disabling OpenCL for this session!"
-           " likely problems:\n"
+           " some possible causes:\n"
            "  - OpenCL out of resources due to preference settings. please try with defaults,\n"
-           "  - buggy driver for some devive. please run darktable with `-d opencl' to identify,\n"
-           "  - some drivers (for small or old devices) don't support proper number of events,\n"
+           "  - buggy driver for some device. please run darktable with `-d opencl' to identify,\n"
+           "  - some drivers (for small memory or old devices) don't support needed number of events,\n"
```

Comment:
1. "some drivers don't support needed numbers of events,\n" would probably better, the "reason" might confuse more than it helps :-)

Reply:
I have translated this part already and didn't even notice typo in `devive`! The brain is incredible at reconstructing information :)

Reply:
> 1. "some drivers don't support needed numbers of events,\n" would probably better, the "reason" might confuse more than it helps :-)

I tend to agree, because the only thing a user can do if there is a problem with the drivers is to update them. Therefore, from a practical point of view, it does not matter whether we described possible driver problems in detail or in less detail.

I removed the fragment in parentheses.

Reply:
> I have translated this part already and didn't even notice typo in `devive`! The brain is incredible at reconstructing information :)

Yes, it's very easy to miss because our brain automatically corrects this when we read to understand the text (yes, our brain creates an _expected_ reality rather than reflects reality). Therefore, to detect typos rather than semantic errors, we need to focus exclusively on the perception of individual words without even trying to understand the text during this process. That's why I sometimes brew coffee to adjust my perception and spend time reading texts in this way. :)